index.html

```diff
@@ -77,26 +77,27 @@ <h2>
                 </section>
 
                 <!-- Projects -->
-                <section class="projects" id="projects">
+                <section class="projects__wrapper" id="projects">
```

Comment:
Tohle je blok, který obsahuje elementy. Pro slovo wrapper není důvod skoro nikdy.

```suggestion
                <section class="projects" id="projects">
```